### ISSUE TITLE
Wanted to Translate the Book

### ISSUE THREAD

**Issue by nilTheDev** [progit/progit2#1681]
This book is precious. Though I noticed there is no translation in my mother tongue - Bengali. So, I have decided to start the work.

What I understood from reading the translation guide is that I have to create a Github organization. That would be titled **progit2-[your code]** where **[your code]** is the **iso 639** code of the language. Upon checking the Wikipedia page I found that **bn** is the **iso 639-1** code. However, there is someone who has already taken this name, i.e **progit2-bn**. The repository doesn't even have any activities.

I was wondering whether I can use the **iso 639-2/T** code **(progit2-ben)**. It is a three-letter code and the title is available on Github.

PS: I am very new to this open-source culture. I could not find any way to contact the maintainers.  I tried mailing @ben using the mail address from the log history. Perhaps that is not the proper way.

**Comment by HonkingGoose** [comment]
I'll ping @jnavila here, as they know a lot about translating the book, and can probably offer advice on the name of your project. 😉

**Comment by nilTheDev** [comment]
> I'll ping @jnavila here, as they know a lot about translating the book, and can probably offer advice on the name of your project.

If I remember correctly I think he contributes to the Italian translation. I was actually following `progit2-it` to understand how to begin. Anyway, thank you for the mention. Hopefully, I would get some suggestions.

**Comment by ben** [comment]
Are those directions even correct these days? It looks like what we _actually_ do is move a repo's ownership to the `progit` org, and set the translators as admins there. Does it matter if the repo starts its life in a correctly-named org or belonging to a user?

**Comment by nilTheDev** [comment]
> Are those directions even correct these days? It looks like what we _actually_ do is move a repo's ownership to the `progit` org, and set the translators as admins there. Does it matter if the repo starts its life in a correctly-named org or belonging to a user?

Apologies, I am not a native English speaker. But what I understand is, effectively, I can choose whatever name I want for the repository.  Am I right?

**Comment by HonkingGoose** [comment]
> Are those directions even correct these days? It looks like what we _actually_ do is move a repo's ownership to the `progit` org, and set the translators as admins there. Does it matter if the repo starts its life in a correctly-named org or belonging to a user?

The new Bengali translation can use whatever name @nilTheDev likes. Once the translation is far enough along, they can transfer ownership of the repository to the `progit` organization. Once the repository is transferred under the `progit` org, we can use the `progit2-bn` repository name.

**Comment by nilTheDev** [comment]
> > Are those directions even correct these days? It looks like what we _actually_ do is move a repo's ownership to the `progit` org, and set the translators as admins there. Does it matter if the repo starts its life in a correctly-named org or belonging to a user?
> 
> The new Bengali translation can use whatever name @nilTheDev likes. Once the translation is far enough along, they can transfer ownership of the repository to the `progit` organization. Once the repository is transferred under the `progit` org, we can use the `progit2-bn` repository name.

Thank you. I got my answer. I am closing this issue.

**Comment by jnavila** [comment]
Hello,

Sorry to be late on this. As HonkingGoose wrote, you can choose any name for your repository. These directions are based on past experience where:

 * people asked that I create a repository, only to not work on translating. So, I let them create their own repo and only ping me when some work is available (I call it the "show me the code" principle).
 * When several people want to work together on a repo, it is always simpler to use a GitHub organization to manager rights and ownership. At some point, I need to have administrative rights on the repo in order to set up an manage automatic compilation and deployment on git-scm.com. If a repo is not alive and a new contributor wishes to help, I can still transfer it to progit organization, although that means taking ownership of the repo and the corresponding management burden.

To sum up, you should start a repo for your translation, then once some sizeable work has been performed ping me and allow me administrative rights.
